crates/nu-cli/src/commands/rm.rs

```diff
@@ -73,27 +73,28 @@ impl WholeStreamCommand for Remove {
     }
 }
 
-fn rm(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStream, ShellError> {
+async fn rm(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStream, ShellError> {
     let registry = registry.clone();
-    let stream = async_stream! {
-        let name = args.call_info.name_tag.clone();
-        let shell_manager = args.shell_manager.clone();
-        let (args, _): (RemoveArgs, _) = args.process(&registry).await?;
-        let mut result = if args.trash.item && args.permanent.item {
-            OutputStream::one(Err(ShellError::labeled_error(
-                "only one of --permanent and --trash can be used",
-                "conflicting flags",
-                name
-            )))
-        } else {
-            shell_manager.rm(args, name)?
-        };
-        while let Some(item) = result.next().await {
-            yield item;
-        }
+    let name = args.call_info.name_tag.clone();
+    let shell_manager = args.shell_manager.clone();
+    let (args, _): (RemoveArgs, _) = args.process(&registry).await?;
+    let mut result = if args.trash.item && args.permanent.item {
+        OutputStream::one(Err(ShellError::labeled_error(
+            "only one of --permanent and --trash can be used",
+            "conflicting flags",
+            name,
+        )))
+    } else {
+        shell_manager.rm(args, name)?
```

Comment:
Same here, if you doing use `?` and then for the case of the if above, returned `Ok(OutputStream...)`